conf/application.conf

```diff
@@ -14,6 +14,9 @@
 
 include "frontend.conf"
 
+# Sets maximum file size for file upload (15MB)
+play.http.parser.maxDiskBuffer = 15728640
```

Comment:
Doesn't this contradict the 2MB limit that's imposed on CSV files in the controller?

Reply:
The 2MB in the controller takes precedence over this value but yeah it would make more sense to keep them aligned